assets/js/staticman.js

```diff
@@ -0,0 +1,100 @@
+(function ($) {
+  $('.page__comments-form').submit(function () {
+    var form = this;
+
+    $(form).addClass('disabled');
+    $('button[type="submit"]:enabled').addClass('hidden'); // hide "submit"
+    $('button[type="submit"]:disabled').removeClass('hidden'); // show "submitted"
+
+    var endpoint = '{{ .endpoint | default "https://staticman-frama.herokuapp.com" }}';
+    var gitProvider = '{{ .gitprovider }}';
+    var repo = '{{ .repo }}';
+    var branch = '{{ .branch }}';
+
+    $.ajax({
+      type: $(this).attr('method'),
+      url: [endpoint, 'v3/entry', gitProvider, repo, branch, 'comments'].join('/'),
```

Comment:
Separated POST request URL in JS code to prevent abuses by spam bots through scraping the generated HTML source code.